app/components/team_sidebar/team_list/team_item/team_icon.tsx

```diff
@@ -101,9 +101,12 @@ export default function TeamIcon({
         return backgroundColor ? [styles.container, {backgroundColor}] : [styles.container, nameOnly && styles.nameOnly];
     }, [styles, backgroundColor, selected, nameOnly]);
 
+    const textTypography = typography('Heading', smallText ? 200 : 400, 'SemiBold');
+    textTypography.fontFamily = 'Metropolis-SemiBold';
```

Comment:
This should not be needed, but I see that for size 400 the SemiBold style is not added... Can you file a ticket to investigate?